lib/cndb/omf.h

```diff
@@ -38,10 +38,11 @@ enum cndb_rec_type {
     CNDB_TYPE_KVS_DEL = 5,
     CNDB_TYPE_KVSET_ADD = 6,
     CNDB_TYPE_KVSET_DEL = 7,
-    CNDB_TYPE_ACK = 8,
-    CNDB_TYPE_NAK = 9,
+    CNDB_TYPE_KVSET_MOVE = 8,
+    CNDB_TYPE_ACK = 9,
+    CNDB_TYPE_NAK = 10,
 
-    CNDB_TYPE_CNT = 9,
+    CNDB_TYPE_CNT = 10,
```

Comment:
I presume that will break existing kvdbs?

Reply:
Yes, it does.